examples/crust_peer.rs

```diff
@@ -300,6 +300,7 @@ fn main() {
     let (peer_id, peer_sk) = new_peer_id();
     let mut service = unwrap!(Service::with_config(event_sender, config, peer_id, peer_sk));
     unwrap!(service.start_listening_tcp());
+    unwrap!(service.set_ext_reachability_test(false));
```

Comment:
Ah OK if it were a test. Since this is an example should we put a sample-config and set it there and allow users to set it accordingly too ? Some might want to use this in different networks where ext-reach. might be wanted by them (even to test).